docs/2023/Sunbird inQuiry/audio-upload-feature/updates/2023-07-28.md

```diff
@@ -4,13 +4,23 @@ author: Simran Nigam
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
-
+- [X] Audio Upload Enhancement Planning
+- [X] Audio upload enhancement Implementation
+- [X] Preparation and presention of Midpoint showcase 
 ## Screenshots / Videos 
+<img width="755" alt="17-06-22" src="https://github.com/Code4GovTech/c4gt-milestones/assets/95395832/3a9abdd7-a68d-4cd0-854e-22747ef006d6"/>
+<img width="755" alt="17-06-22" src="https://github.com/Code4GovTech/c4gt-milestones/assets/95395832/a83b6ee8-cd1e-4f8c-9132-c576adf4d55e"/>
```

Comment:
Alternate recommendation instead of image tags you could use ![altText](linkToImage)